### PR TITLE
dart lint, removed api key, fixed error in text scaffold

### PR DESCRIPTION
**Pull request by oxyfalcon**
* Changed the workflow by adding flutter pub get and upgrade
* Added flutter analyze for static compile time errors
* Fixed the minor issue in the  #15 
* @skfareeduddin Please check the code once for future reference. Also do be careful in the future about the static errors

### CODE REVIEW

**Comment by oxyfalcon** [comment]
@bonamiAkash 
Please check the code once

**Comment by bonamiAkash** [comment]
Why did you remove the google files @oxyfalcon

**Comment by oxyfalcon** [comment]
@bonamiAkash Making sure the api key doesn't get exposed. Besides I think it is better if the people who want to contribute to the project setup their own firebase console.

**Comment by bonamiAkash** [comment]
Makes sense then

**Comment by bonamiAkash** [comment]
Also it is a good thing that you added in the flutter analyse command in the workflow @oxyfalcon 
Good lesson to be learnt here.

File: .github/workflows/dart.yml [hunk 1]
```diff
@@ -19,12 +19,21 @@ jobs:
...
     - name: Lint
       run: dart format --output=none --set-exit-if-changed .
-
-    - name: Pub Upgrade
-      run: flutter pub upgrade
+    
+    - name: dart anaylze
+      run: dart analyze
+    
+    - name: flutter analyze
```
Comment (bonamiAkash) [inline .github/workflows/dart.yml:35]: @oxyfalcon Do we need both flutter and dart analyse commands. Shouldn't one be sufficient

**Review by bonamiAkash — CHANGES_REQUESTED** [review]

File: .github/workflows/dart.yml (hunk 1 above)
Comment (oxyfalcon) [reply .github/workflows/dart.yml:35]: @bonamiAkash dart analyse is mainly for the packages and the flutter analyse is mainly for the app level checks
Please check this link in stackOverflow -> [https://stackoverflow.com/questions/65308789/why-is-flutter-analyze-different-from-dart-analyze](url)

Comment (bonamiAkash) [reply .github/workflows/dart.yml:35]: @oxyfalcon ok
I guess this is fine then.

**Review by bonamiAkash — APPROVED** [review]
LGTM!

**Pull request merged**